specifications/android.md

```diff
@@ -49,18 +61,22 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
 
-The connector send this action to register to push messages. The intent MUST contain 2 String extras:
-* application: with the end user application package name. The distributor MUST be able to handle many connections with a single application.
-* token: with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+The connector sends this action to register to push messages. The intent MUST contain 2 extras:
+* application (String): the end user application package name. The distributor MUST be able to handle many connections with a single application.
+* token (String): a randomly generated token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+
+It MAY be sent with the following 2 extras:
+* features (ArrayList<String>): indicate the connector is requesting a set of optional features to be enabled. It MUST be the qualified name of the action declared to advertise this feature. The connector MUST check that the action is declared before requesting an optional feature.
+* message (String) : a short description of the purpose of the registration that the connector MAY show to the user.
```

Comment:
1. Remove space in front of the colon 
2. Should this be "that the distributor MAY  show to the user"?

Reply:
indeed, thanks !